core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedClusterMessage.scala

```diff
@@ -92,6 +94,11 @@ private[spark] object CoarseGrainedClusterMessages {
       hostToLocalTaskCount: Map[String, Int])
     extends CoarseGrainedClusterMessage
 
+  // Check if an executor was force-killed but for a normal reason
+  // This could be the case if e.g. the cluster manager supports killing an executor to
+  // move it elsewhere or to kill an executor in order to free resources
```

Comment:
I would just say `This could happen if the executor is preempted, for instance.`